src/Castle.Facilities.EventWiring/Castle.Facilities.EventWiring-VS2017.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net45</TargetFramework>
+   
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Castle.Facilities.EventWiring</AssemblyName>
+    <RootNamespace>Castle.Facilities.EventWiring</RootNamespace>
+    <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+    <SignAssembly>true</SignAssembly> 
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);DOTNET45</DefineConstants>
```

Comment:
I assume something the @fir3pho3nixx's common.props file will come later since DefineConstants is undefined until defined, or do I misunderstand how this works?

Also is there a reason all these projects except the main Windsor project define DOTNET45?